### DISCUSSION TITLE
How to use MatchesJsonPath

### DISCUSSION THREAD

**Discussion by Jadeclon** [HBOCodeLabs/wiremock-captain#174] [Q&A] [answered]
Can someone tell how to use MatchesJsonPath to ensure specific json attributes are send?

     
            const request2: IWireMockRequest = {
                method: 'POST',
                endpoint: privateApi + 'test',
                body: {"matchesJsonPath": "country"}
            };

            const mockedResponse2: IWireMockResponse = {
                status: 200,
                body: {
                    msg:'Post Respond'
                },
            };

            const features: IWireMockFeatures = {
                requestBodyFeature: MatchingAttributes.MatchesJsonPath,
            };

            await mock.register(request2, mockedResponse2, features);

**Comment by satvik-s** [comment, marked as answer]
Hi @Jadeclon!

I would suggest referencing this: https://wiremock.org/docs/request-matching/

To summarize, if you use `$.country` as the input to `matchesJsonPath` then the request will be a match iff the input body has a key called `country`
